src/verilog.y

```diff
@@ -1621,6 +1625,7 @@ interface_declaration:          // IEEE: interface_declaration + interface_nonan
                         { if ($2) $1->addStmtsp($2);
                           if ($3) $1->addStmtsp($3);
                           if ($5) $1->addStmtsp($5);
+                          GRAMMARP->m_modp = nullptr;
```

Comment:
Modules can be inside modules so this breaks.  It needs to be pushed/popped. Perhaps the SYMP tracking can be used instead (have it search back for the current module)?

Reply:
Yes, the statement in end of class that inside module corrupts m_modp of the parent module. And it seems to be little bit difficult to avoid the use of the m_modp variable. So probably must use a stack for it.

Thanks to the SYMP so we can reuse the information.